File: Core/Entities/Structure/Issue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Core.Entities.Structure
{
	public class Issue : CoreEntity
	{
		public String IssueTitle { get; set; }
		public String IssueDescription { get; set; }
		public int Severity { get; set; }
		public IssueType IssueType { get; set; }

		public ICollection<IssueComments> Comments { get; set; } = new List<IssueComments>();
	}
}

File: client/src/app/entities/structure/issue.ts
import { IIssueComment } from './issuecomment';
import { IIssueType } from './issuetype';

export interface IIssue {
  guid: string;
  issueTitle: string;
  issueDescription: string;
  severity: number;
  issueType: IIssueType;
  comments: IIssueComment[];
}
